assets/filter_repos.js

```diff
@@ -0,0 +1,37 @@
+const searchInput = document.getElementById('search-hook-id');
+const selectInput = document.getElementById('select-hook-type');
+
+const hooks = document.getElementById('hooks');
+const repos = hooks.getElementsByTagName('ul');
```

Comment:
this sets all of these as globals -- this should be wrapped in an anonymous function (+ `'use strict';`)